Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -170,7 +170,7 @@ class HistoryPanel: SiteTableViewController, HomePanel {
         let refresh = UIRefreshControl()
         refresh.addTarget(self, action: #selector(HistoryPanel.refresh), for: UIControlEvents.valueChanged)
         self.refreshControl = refresh
-        self.tableView.addSubview(refresh)
+        self.tableView.refreshControl = refresh
```

Comment:
we should make sure that `removeRefreshControl` works correctly. Right now it is doing 
`        self.refreshControl?.removeFromSuperview()`
it probably needs to also do
`self.table.refreshControl = nil`

Reply:
Thanks for pointing that out.  Please review again.